spec/models/miq_schedule_spec.rb

```diff
@@ -37,19 +41,67 @@
     end
 
     context "SmartState" do
+      let(:sched_action) { { :method => "vm_scan", :options => options } }
 
+      it "exports to array" do
+        miq_schedule_array = MiqSchedule.export_to_array([miq_schedule.id], MiqSchedule).first["MiqSchedule"]
+        expect(miq_schedule_array.slice(*MiqSchedule::ImportExport::SKIPPED_ATTRIBUTES)).to be_empty
+        expect(miq_schedule_array['sched_action'][:method]).to eq("vm_scan")
+        expect(miq_schedule_array['filter_resource_name']).to eq(miq_report.name)
```

Comment:
@d-m-u it would be nice also add miq_search and with MiqSearch yaml looks:


```ruby
---
- MiqSchedule:
    name: miqSearch
    description: miqSearch
    sched_action:
      :method: vm_scan
    filter: 
    resource_type: Vm
    run_at:
      :tz: UTC
      :start_time: 2019-08-28 00:00:00.000000000 Z
      :interval:
        :unit: once
        :value: ''
    enabled: true
    userid: admin
    prod_default: 
    miq_search_id: 7
    adhoc: 
    file_depot_id: 
    resource_id: 
    MiqSearchContent:
    - MiqSearch:
        name: default_Status / Running
        description: Status / Running
        options: 
        filter: !ruby/object:MiqExpression
          exp:
            and:
            - "=":
                field: Vm-power_state
                value: 'on'
            - "=":
                field: Vm-active
                value: 'true'
        db: Vm
        search_type: default
        search_key: 

```

Reply:
you need to test field `miq_search_id ` and `MiqSearchContent`